Makefile

```diff
@@ -86,7 +87,7 @@ run: manifests
 .PHONY: container-build
 container-build: CGO_ENABLED=0
 container-build:
-	ko build --push=false $(KO_OPTS) $(ARGS) .
+	KO_DOCKER_REPO="$(IMAGE_HOST)/$(IMAGE_NAMESPACE)" GOFLAGS="${GOFLAGS} ko build --push=false $(KO_OPTS) $(ARGS) .
```

Comment:
Same as the previous comment mentioning `.EXPORT_ALL_VARIABLES` behavior.

Reply:
fixed

Reply:
The variables `KO_DOCKER_REPO` and `GOFLAGS` are already exported globally (i.e.: `.EXPORT_ALL_VARIABLES` target). Please remove them from the local target.